### PR TITLE
fix: fixed icon of vue

### PR DESCRIPTION
**Pull request by jfbeatit**
Fix reason: I use the font `SauceCodePro Nerd Font Mono `, after i update this plugin to the latest version, the vue icon can not display anymore.

### CODE REVIEW

**Comment by gegoune** [comment]
Try updading your font instead, this icon has been recently changed to more accurate one.

**Comment by alex-courtis** [comment]
Let's merge #264 first

**Comment by gegoune** [comment]
Not sure we want to merge this one, Alex. Vue icon has been very recently updated to one that's reverted here.

If we are using proper icons that are only available in recent font versions we will endlessly get requests like this while solution is to update fonts. 

We will shortly provide compat tag for version 2 icons. That might solve the issue this pr tries to address.

Or we should come up with proper policy on how we handle those things.

File: lua/nvim-web-devicons.lua [hunk 1]
```diff
@@ -1453,7 +1453,7 @@ local icons_by_file_extension = {
     name = "Vim",
   },
   ["vue"] = {
-    icon = "",
+    icon = "﵂",
```
Comment (alex-courtis) [inline lua/nvim-web-devicons.lua:1456]: Unfortunately this is a nerd font v2 icon; we need a v3 icon.

See #264

You'll need to run nerdfix.

**Review by alex-courtis — CHANGES_REQUESTED** [review]

**Comment by alex-courtis** [comment]
> Not sure we want to merge this one, Alex. Vue icon has been very recently updated to one that's reverted here.
> 
> If we are using proper icons that are only available in recent font versions we will endlessly get requests like this while solution is to update fonts.

It looks like the vue icon on master is the correct v3 icon.

> We will shortly provide compat tag for version 2 icons. That might solve the issue this pr tries to address.
> 
> Or we should come up with proper policy on how we handle those things.

The tag should be enough.

I have Hack Nerd Font Mono set as my browser font, so I can quickly see in the PR whether the icon is a valid (v3) icon.

**Comment by alex-courtis** [comment]
> Or we should come up with proper policy on how we handle those things.

We _could_ maintain a v2 branch rather than a tag, for reverting v3->v2 icons.

That's a lot of work and the branches will completely diverge.

**Comment by alex-courtis** [comment]
Optimistically closing: @jfbeatit can you please update your font to v3 and test again.

We'll reopen if that doesn't work.

**Pull request closed without merging**